cache/search_stats.go

```diff
@@ -506,7 +506,7 @@ func (ssc *SearchStatsCacheImpl) loadSourcesByPositionAndParent() (map[string]st
 	from sources p
 		join sources c on c.parent_id = p.id
 		join sources gc on gc.parent_id = c.id
-	where c.position is not null and p.name = 'TES'
+	where c.position is not null and p.uid = 'xtKmrbb9'
```

Comment:
Just add a comment please that will describe what we are doing and why.  Also mention in the comment that 'xtKmrbb9' is the uid of TES.